configs/rover_configs.yaml

```diff
@@ -15,4 +15,21 @@
 # limitations under the License.
 #
 
-core:
\ No newline at end of file
+core:
+  backend:
+    # The backend server address
+    addr: ${ROVER_BACKEND_ADDR:localhost:11800}
+    # The TLS switch
+    enable_TLS: ${ROVER_BACKEND_ENABLE_TLS:false}
+    # The file path of client.pem. The config only works when opening the TLS switch.
+    client_pem_path: ${ROVER_BACKEND_PEM_PATH:"client.pem"}
+    # The file path of client.key. The config only works when opening the TLS switch.
+    client_key_path: ${ROVER_BACKEND_KEY_PATH:"client.key"}
```

Comment:
I think these two should remove `client_` suffix too.

Reply:
These two key is works for the TLS  mode in client config, I think we keep it is more better.